packages/web/src/Spaces.tsx

```diff
@@ -120,12 +120,14 @@ export function Spaces() {
         </PatternFlyPopup>
       )}
 
-      <PageSection variant={PageSectionVariants.light}>
+      <div className="pf-c-page__main-nav">
         <Breadcrumb>
           <BreadcrumbItem>
             <Link to={routes.spaces()}>Spaces</Link>
           </BreadcrumbItem>
         </Breadcrumb>
+      </div>
+      <PageSection variant={PageSectionVariants.light} className="pf-u-pt-sm">
```

Comment:
Same question here.. With the React components PatternFly 4 provides, seems odd that we need to "manually" add classes every now and then.

Reply:
This is a utility helper class to adjust the spacing. My hope is that as the PF4 Page component is refined, this will be handled there.